app/src/main/res/values/strings.xml

```diff
@@ -488,4 +488,15 @@
     <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog.</string>
     <string name="preference_accessibility_text_size_summary">Make text on websites larger or smaller</string>
     <string name="preference_accessibility_font_size_title">Font Size</string>
+
+
+    <!-- Onboarding theme -->
+    <!-- Automatic theme setting (will follow device setting) -->
+    <string name="onboarding_theme_automatic_title">Automatic</string>
```

Comment:
Can you put these in `static_strings.xml` with translatable false? I'm not sure all the copy is 100% for the onboarding cards :)